test-crates/bugreport-client-with-git-hash-feature/tests/integration_tests.rs

```diff
@@ -0,0 +1,29 @@
+use bugreport_test_crate_utils::assert_bin_stdout;
+
+#[test]
+fn with_git_repo() {
+    assert_bin_stdout(
+        "bugreport-client-with-git-hash-feature",
+        r"#### Software version
+
+bugreport-client-with-git-hash-feature 0.1.0 \([0-9a-f]{7,}(-modified)?\)
```

Comment:
Super nitpicky: the dots in 0.1.0 should be escaped.

Reply:
Nit-picky but well-spotted. Will fix!